pyquil/api/_qpu.py

```diff
@@ -69,12 +69,14 @@ def _extract_bitstrings(ro_sources: List[Optional[Tuple[int, int]]],
 
 class QPU(QAM):
     @_record_call
-    def __init__(self, endpoint: str, user: str = "pyquil-user") -> None:
+    def __init__(self, endpoint: str, user: str = "pyquil-user", priority: int = 1) -> None:
         """
         A connection to the QPU.
 
         :param endpoint: Address to connect to the QPU server.
         :param user: A string identifying who's running jobs.
+        :param priority: The priority with which to insert jobs into the Lodgepole queue. Lower
```

Comment:
Maybe replace "Lodgepole" with "QPU".

Reply:
Also I wonder whether it would be better to also have this optional to `QPU.run()`: a user would not need to create a new QPU object to schedule a job with a different priority.

Reply:
Agree with Mark